checks/runtimeCheck_test.go

```diff
@@ -0,0 +1,25 @@
+package checks
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRuntimeCheck(t *testing.T) {
+	dirname, _ := os.Getwd()
+	runtimeLookupPath = fmt.Sprintf("%s/%s", dirname, runtimeLookupPath)
+	os.MkdirAll(runtimeLookupPath+"/node", os.ModePerm)
+	defer os.RemoveAll(dirname + "/var")
```

Comment:
Can we make this path more specific? Something like `/var/task/...` or something like that?